### PR TITLE
Changes for Kitura-Bench to use public git repo of Kitura-Benchmarks

### PR DESCRIPTION
**Pull request by mamabusi**

### CODE REVIEW

File: Bench-Kitura-Blog/build.sh [hunk 1]
```diff
@@ -7,8 +7,7 @@
 set -e
 
 # Location of this script
-dir=$($(cd `dirname $0`) && pwd)
-
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (djones6) [inline Bench-Kitura-Blog/build.sh]: Nit-pick ... blank line was deleted

File: Bench-Kitura-Core/latest/Package.swift [hunk 2]
```diff
@@ -19,10 +19,10 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.2.0"),
-        .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", from: "1.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura-Session.git", from: "3.0.0"),
-        .package(url: "https://github.com/IBM-Swift/Kitura-Compression.git", from: "2.1.1"),
+        .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.4.0"),
```
Comment (djones6) [inline Bench-Kitura-Core/latest/Package.swift]: Do you think that changing the minimum minor/patch versions is necessary?  For example, there's nothing in these benchmarks (yet) that would require Kitura 2.4, they would compile fine on 2.2 or 2.3.  I'd prefer to update these if and when we introduce benchmarks that use the new APIs.

**Review by djones6 — CHANGES_REQUESTED** [review]
I think it's only the 'Location of this script' fix required here (I also tripped over this while implementing #3, but I'm happy to merge this one and rebase mine).

I don't think we want to update the minimum package minor/patch levels (unless I've missed something), because there's nothing in the benchmarks that requires the newer versions, and SPM should pick the latest available versions anyway.

**Comment by mamabusi** [comment]
@djones6 I have made the requested changes. 
I am fine with you taking the call on whether to put this fix as part of the #3 or merging it as a separate PR. This change would be required for my other PR on Kitura-Bench. Thank you.

**Review by djones6 — APPROVED** [review]

**Pull request merged**